templates/zerver/help/deactivate-or-reactivate-a-user.md

```diff
@@ -28,6 +28,33 @@ user's bots will also be deactivated. Lastly, the user will be unable to
 create a new Zulip account in your organization using their deactivated
 email address.
 
+### Notify users of their deactivation
+
+If you want to notify the user of their deactivation,
+you can do so by sending them a email.
+
+{start_tabs}
+
+{settings_tab|user-list-admin}
+
+ 2. Click the **Deactivate** button to the right of the user account that you
+want to deactivate.
+
+ 3. Check the checkbox labeled **"Send a notification email to \[user]?"**.
+
+ 4. Enter your message for the user in the appearing textbox.
+
+ 3. Approve by clicking **Confirm**.
+
+{end_tabs}
+
+Zulip will then, in addition to deactivating the user, send an email
+containing your message to the users email address.
+
+Below you can see how the email that the user receives will look like:
+
+<img src="/static/images/help/deactivate-user-email.png" alt="view-of-admin" width="600"/>
```

Comment:
We should hold off on regenerating the screenshot until we've finalized all the text details, but we should leave off the address footer at the bottom of the email, since the fake address used in the development environment is kinda goofy :)

Reply:
OK, we're ready to do this @juliaBichler01.